### ISSUE TITLE
Default 

### ISSUE THREAD

**Issue by kaitlyngaynor** [zooniverse/Panoptes-Front-End#5782]
For camera trap projects, specifically [WildCam Gorongosa](https://www.zooniverse.org/projects/zooniverse/wildcam-gorongosa/classify), there is an option to animate a loop between multiple images in a sequence. We would prefer this to be the default presentation (the animation could be shown 2-3 times upon first appearance), as many users are unaware that this option exists, and end up only looking at the first image in the sequence and making classifications based on this image, which biases our data.

At the least, it would be helpful to move the animation button to the right side of the image closer to the species selection pane, as now you have to move the mouse all the way to the left of the screen to turn on the animation, or have a keyboard shortcut for playing the animation.

See discussion from users on [the chat forum](https://www.zooniverse.org/projects/zooniverse/wildcam-gorongosa/talk/79/1536360?comment=2563867&page=1).

**Comment by njsgastro** [comment]
In addition to the loop change request  by Kaitlingaynor above,  a image brightness control would be extremely helpful since many images  contain deep shadows making it difficult to identify what  animal is hiding there.

**Comment by eatyourgreens** [comment]
Auto-playing animation and video can be an accessibility barrier (see [WCAG Criterion 2.2.2](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html)) but I think that's fine as long as volunteers have an option to disable autoplay globally in their user profile. Anonymous users could, perhaps, set a cookie that turns off autoplay if they find it distracting (or if it induces nausea.)

Another suggested accessibility technique is to [play animations for 5s or less](https://www.w3.org/WAI/WCAG21/Techniques/general/G11), in order to minimise distraction or discomfort. I think we already limit the number of times a loop will play, once started.

**Comment by kaitlyngaynor** [comment]
Makes sense. I think it would be best if users could turn this feature off (with cookies or in a profile). And yes, the loop only plays 3 times currently, once the button is pressed.

**Comment by njsgastro** [comment]
I know it's been 2 years since this issue has been opened. However, am curious, if any consideration has been given to this suggestions?  Doing a fair amount of programming myself, I don't see where these suggestions present a huge undertaking.
